### ISSUE TITLE
Joystick Support

### ISSUE THREAD

**Issue by calejvaldez** [ojosproject/iris#44]
### Summary

Iris should be able to support joysticks for accessibility purposes.

### Concept images and/or code

[gilrs](https://docs.rs/gilrs/latest/gilrs/)

**Comment by calejvaldez** [comment]
A way I think we can approach this is by creating a `JoystickWrapper` component that takes props labeled `left`, `right`, `up`, `down` and would include an id, like `{left: "COMPONENT_ID"}` . When a user makes a movement using a joystick, the `JoystickWrapper` would have a red border. If a user makes a valid movement (that is, a key is available in the object for that movement), it'll add a red border to the `COMPONENT_ID`.

Pseudocode:

```tsx
type JoystickWrapperProps = {
  left?: string;
  right?: string;
  up?: string;
  down?: string;
}

export default function JoystickWrapper({left, right, up, down}) {
   
}
```
